packages/graphql/src/classes/Neo4jGraphQL.ts

```diff
@@ -287,11 +288,18 @@ class Neo4jGraphQL {
         };
 
         const resolversComposition = {
-            "Query.*": [wrapResolver(wrapResolverArgs)],
-            "Mutation.*": [wrapResolver(wrapResolverArgs)],
-            "Subscription.*": [wrapSubscription(wrapResolverArgs)],
+            "Query.*": [wrapQueryAndMutation(wrapResolverArgs)],
+            "Mutation.*": [wrapQueryAndMutation(wrapResolverArgs)],
         };
 
+        if (this.features.subscriptions) {
+            resolversComposition["Subscription.*"] = wrapSubscription({
+                subscriptionsEngine: this.features.subscriptions,
+                schemaModel: this.schemaModel,
+                authorization: this.authorization,
+            });
+        }
```

Comment:
@angrykoala, with this conditional, we can remove the check we discussed from the wrapper, which I think improves the comprehensibility of the code. 🙂